imagersite/imagersite/views.py

```diff
@@ -1,12 +1,16 @@
 from django.views.generic import TemplateView
 from photo.models import Photos
 
-
 class HomeView(TemplateView):
     template_name = 'home.html'
 
     def get_context_data(self, **kwargs):
         context = super(HomeView, self).get_context_data(**kwargs)
-        context["photo"] = Photos.objects.filter(published='public')\
-            .order_by('?').first()
+        try:
+            context["photo"] = Photos.objects.filter(published='public')\
+                .order_by('?').first()
+            # returns pic of dog if above is context is false/att error
+        except AttributeError:
+            context["photo"] = 'imagersite/media/photo_files/15-07-28/dog.jpg'
```

Comment:
be a bit cautious here.  In the first case, `context["photo"]` will be a photo object.  In the second, it's a string.  These are not the same thing.  You want to pass the same types of values back from this function so you can use the `photo` symbol in your templates in the same way regardless.  

Also, your url for this stock photo probably ought to be one that is related to the media_url, not the media root.  this url is unlikely to work once django is deployed.